docker-compose.yml

```diff
@@ -33,15 +33,6 @@ services:
       - 8080:80
       - 443:443
       - 9500:8080
-    ulimits:
```

Comment:
Why are we removing this as part of this PR?

Reply:
It's something that is not supposed to be here anyway.

Reply:
Can you explain? Where should it be in that case?

Reply:
It breaks Gitpod workspaces.